kit/azure/bootstrap/main.tf

```diff
@@ -1,10 +1,9 @@
 data "azuread_client_config" "current" {}
 
-data "azurerm_management_group" "root" {
-  name = var.aad_tenant_id
-}
+data "azurerm_subscription" "current" {}
 
-data "azurerm_subscription" "current" {
+data "azurerm_management_group" "root" {
+  name = data.azuread_client_config.current.tenant_id
```

Comment:
d: can also use azurerm_subscription.current.tenant_id for this, no need for azuread_client